Makefile

```diff
@@ -145,13 +149,11 @@ yq:
 
 # Generate bundle manifests and metadata, then validate generated files.
 .PHONY: bundle
-bundle: operator-sdk manifests yq
+bundle: operator-sdk manifests yq csv-generator
 	./operator-sdk generate kustomize manifests -q
+	./csv-generator --csv-version $(VERSION) --operator-image $(IMG) --operator-version $(VERSION) > bundle/manifests/ssp-operator.clusterserviceversion.yaml
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	$(KUSTOMIZE) build config/manifests | ./operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
-	./yq w -i bundle/manifests/ssp-operator.clusterserviceversion.yaml spec.webhookdefinitions[0].containerPort 9443
```

Comment:
The functionality of these lines should be added to the csv-generator, requried by OLM

Reply:
That seems to be there already, the CSV yaml coming out of csv-generator is virtually identical to the one from the latest release. (though the csv-generator code doesn't grep for "9443"...)